Experimental/src/org/sleuthkit/autopsy/experimental/configuration/AutoIngestUserPreferences.java

```diff
@@ -267,17 +258,35 @@ public static void setMaxNumTimesToProcessImage(int retries) {
         ModuleSettings.setConfigSetting(UserPreferences.SETTINGS_PROPERTIES, MAX_NUM_TIMES_TO_PROCESS_IMAGE, Integer.toString(retries));
     }
 
-     /**
+    /**
+     * Get maximum number of concurrent ingest nodes allowable for one case at a
+     * time.
+     *
+     * @return maximum number of concurrent nodes for one case. Default is 3.
+     */
+    public static int getMaxConcurrentJobsForOneCase() {
+        String value = getPreferenceValue(MAX_CONCURRENT_NODES_FOR_ONE_CASE);
+        return value.isEmpty() ? 3 : Integer.parseInt(value);
+    }
+
+    /**
+     * Get maximum number of concurrent ingest nodes allowable for one case at a
+     * time.
+     *
+     * @param numberOfNodes the number of concurrent nodes to allow for one case
+     */
+    public static void setMaxConcurrentIngestNodesForOneCase(int numberOfNodes) {
+        ModuleSettings.setConfigSetting(UserPreferences.SETTINGS_PROPERTIES, MAX_CONCURRENT_NODES_FOR_ONE_CASE, Integer.toString(numberOfNodes));
+    }
+
+    /**
```

Comment:
This attempt at load balancing is removed for this release. I'm guessing either this preference was missed in that process or your branch is slightly out of synch.